pkg/resourcemanager/controller/managedresource/reconciler.go

```diff
@@ -524,6 +525,15 @@ func (r *Reconciler) applyNewResources(ctx context.Context, log logr.Logger, ori
 
 			return fmt.Errorf("error during apply of object %q: %s", resource, err)
 		}
+
+		switch operationResult {
+		case controllerutil.OperationResultCreated:
+			resourceLogger.Info("Created resource because it was not existing before")
+		case controllerutil.OperationResultUpdated:
+			resourceLogger.Info("Updated resource because its actual state differed from the desired state")
+		case controllerutil.OperationResultNone:
+			resourceLogger.V(1).Info("Resource was neither created or updated because its actual state matches with the desired state")
```

Comment:
Nit
```suggestion
			resourceLogger.V(1).Info("Resource was neither created nor updated because its actual state matches with the desired state")
```